js/src/product-feed/product-statistics/index.scss

```diff
@@ -35,25 +35,40 @@
 			}
 		}
 
+		.components-flex {
+			align-items: flex-start;
+		}
+
 		.components-flex + .components-flex {
 			margin-top: $grid-unit-10;
 		}
 
 		.components-flex__item {
 			color: $gray-900;
+			padding-top: 0.25 * $grid-unit;
 
 			&:first-child {
 				min-width: 130px;
 			}
 
-			&:nth-child(2) {
-				display: inline-flex;
-				align-items: center;
-				fill: currentColor;
-				color: #008a20;
+			&.gla-status-icon {
+				min-width: 30px;
+				padding-top: 0;
```

Comment:
Not sure about that. (Or maybe I didn't understand the strategy) 

If I do middle alignment it looks like this on multiline:

<img width="800" alt="Screenshot 2022-04-18 at 16 03 40" src="https://user-images.githubusercontent.com/5908855/163805723-98515b15-25d5-46c3-af6e-c68de5200b1c.png">

Reply:
Oh sorry, I missed that requirement. 🙏  It will need a lot of adjustments to align a larger icon vertically to the first line of multi-line text. I think the current implementation is better.